collector/collector/__init__.py

```diff
@@ -29,11 +29,12 @@ def configure_app(config_object, app):
 app.config.from_object(config.Config)
 
 try:
-   #try importing from the local dev configuration if it exists
-   from . import config_local
-   app.config.from_object(config_local.Config)
-except:
-   pass
+    # try importing from the local dev configuration if it exists
+    from . import config_local
+    app.config.from_object(config_local.Config)
+except Exception as e:
+    print(e)
```

Comment:
PEP8 does not add anything just prints warnings.  I added for debugging purposes, because I assumed config_local will be used for debugging.  I don't have strong opinions about it so I can remove the print if you feel this print is unnecessary.

Reply:
On second thought, I think it is a good idea to keep the print, because it hints that config_local does not exist, and config_local should never be used in production.